go/websocketbridge/features/clientToServer.feature

```diff
@@ -0,0 +1,30 @@
+Feature: 
+
+  As an Exosphere developer
+  I want to be able to send messages from a web client to my Exosphere application
+  So that I can create a useful application
+
+  Rules
+  - If a message does not have a SessionId, a new one is generated for that user
```

Comment:
```
- A new session id is created for each websocket connection
```
Would be good to add a test that the if a second client connects, it has a different session id then the first